pom.xml

```diff
@@ -27,6 +27,7 @@
     <ramlfiles_path>${basedir}/ramls</ramlfiles_path>
     <postgres-testing.version>33.0.0</postgres-testing.version>
     <minio.version>8.3.0</minio.version>
+    <generate-marc-utils-version>1.4.0-SNAPSHOT</generate-marc-utils-version>
```

Comment:
Since the library with required changes wasn't released yet we have to specify the SNAPSHOT version to have latest library changes available